eox_nelp/urls.py

```diff
@@ -29,4 +29,5 @@
         include('eox_nelp.course_experience.frontend.urls', namespace='course-experience-frontend'),
     ),
     path('api/stats/', include('eox_nelp.stats.api.urls', namespace='stats-api')),
+    path('stats/', include('eox_nelp.stats.urls', namespace='stats')),
```

Comment:
Here we have to discuss which would be the default path. I like the prefix fronted cause as a user I relate that it has react or some js extra.
https://github.com/eduNEXT/eox-nelp/pull/57/files#diff-99d9cbe815961e1cc232cd5191394ac2e794e5ea823be05f5f9addf7331a39a7R28

Reply:
That's right the react components should be in the frontend and that's all, if I didn't include the view, inside the frontend folder, is  because a Django view is not a frontend element and that logic is not js or css and finally to add the path frontend in the url is redundant a view is an API or belongs to the presentation layer and you probably won't find  the path frontend in other views